templates/single-tapestry.php

```diff
@@ -72,4 +103,17 @@
     </main><!-- #main -->
 </div><!-- #primary -->
 
-<?php get_footer(); ?>
+<?php
+if ($iframe_mode):
+?>
+
+</div><!-- #content -->
+</div><!-- #page -->
+<?php wp_footer(); ?>
+</body>
+</html>
+
+<?php
+else: get_footer();
+endif;
+?>
```

Comment:
Same as above, I believe you should be able to use the get_footer function to return an alternate footer.